base/doc/ltnews41.tex

```diff
@@ -637,85 +788,106 @@ \subsection{Refinement of \cs{MakeTitlecase}}
 We introduced \cs{MakeTitlecase} as a late addition to the June 2022
 release, making use of the improved case code in \pkg{expl3}. Unlike
 upper and lowercasing, making text titlecased is more tricky to get
-right: this can apply either to the whole text or on a word-by-word
+right: this can be applied either
+to the whole text, or on a word-by-word
 basis.
 
-A subtle issue was reported against the \pkg{expl3} repository
-(\url{https://github.com/latex3/latex3/issues/1316}) which links to
-how we deal with the question of case changing \enquote{words} but
-shows up if you titlecase text stored in a command.
+A subtle issue was reported concerning the \pkg{expl3} repository
+(\url{https://github.com/latex3/latex3/issues/1316}); this
+is related to 
+how we deal with the case changing of \enquote{words} but it also
+shows up when you titlecase some text stored in a command.
 
-We have looked again at how to implement \cs{MakeTitlecase} to be as
-predictable as possible, and have made a change in this release. The
+We have looked again at how to implement \cs{MakeTitlecase} in order to make it as
+predictable as possible, and we have made a change in this release. The
 command no longer tries to lowercase text before applying titlecasing,
-and gives the correct result for text stored in commands.
+and it therefore gives correct results for text stored in commands.
 
+\begin{old}
 We have also added an additional key to the optional argument to
 \cs{MakeTitlecase} which allows the user to decide if this will apply
 only to the first word (the default) or to all words.
+\end{old}
 
-\subsection{Tab character as a special}
+\marginpar{Joseph: rewrite right or wrong?}
+
+We have also added an additional key to the optional argument to
+\cs{MakeTitlecase} which allows the user to decide if the case change gets applied 
+only to the first word (the default) or to all the words.
+
+
+\subsection{Tab character as a special character}
+
+In \LaTeX{} News~38, we described a change to \cs{verb}, etc., that makes
+the tab character equivalent to a space; we have now completed this work by
+adding the tab character to the 
+%% standard ??
```

Comment:
Yeah, could add that

Reply:
> Yeah, could add that

no Chris took it out (you had it :-) and I think he is right "standard" is not helpful at this point